src/qt/rpcconsole.cpp

```diff
@@ -731,8 +731,12 @@ void RPCConsole::clear(bool clearHistory)
         );
 
     message(CMD_REPLY, (tr("Welcome to the %1 RPC console.").arg(tr(PACKAGE_NAME)) + "<br>" +
-                        tr("Use up and down arrows to navigate history, and <b>Ctrl-L</b> to clear screen.") + "<br>" +
-                        tr("Type <b>help</b> for an overview of available commands.")) +
+#ifdef Q_OS_MAC
+                        tr("Use up and down arrows to navigate history, and <b>⌘cmd-L</b> to clear screen.") + "<br>" +
+#else
+                        tr("Use up and down arrows to navigate history, and <b>Ctrl-L</b> to clear screen.") + "<br>" +		
+#endif
+			tr("Type <b>help</b> for an overview of available commands.")) +
```

Comment:
Possible suggestion:
```c++
#ifdef Q_OS_MAC
QString clearKeyStr = "<b>⌘cmd-L</b>";
#else
QString clearKeyStr = "<b>Ctrl-L</b>";
#endif
...
tr("Use up and down arrows to navigate history, and %1 to clear screen.").arg(clearKeyStr) + "<br>" +
```
After all:
- The key is not to be translated
- Also gets rid of the HTML formatting in the translation string in one go